pgjdbc/src/test/java/org/postgresql/test/jdbc2/optional/OptionalTestSuite.java

```diff
@@ -16,13 +16,13 @@
  */
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-        BaseDataSourceFailoverUrlsTest.class,
-        CaseOptimiserDataSourceTest.class,
-        ConnectionPoolTest.class,
-        PoolingDataSourceTest.class,
-        SimpleDataSourceTest.class,
-        SimpleDataSourceWithSetURLTest.class,
-        SimpleDataSourceWithUrlTest.class,
+    BaseDataSourceFailoverUrlsTest.class,
+    CaseOptimiserDataSourceTest.class,
```

Comment:
The arrayInitIndent is 2 and basicOffset/lineWrapping is 4, thus it should follow either arrayInitIndent or basicOffset or lineWrappingIndent. Thus the error is thrown by checkstyle, since it doesn't follow any of these offset Indents.

Reply:
@Abhishek-kumar09 , is this update for https://github.com/checkstyle/checkstyle/pull/8997 ?
it is not merged yet, but waiting for final review.